docs/content/getting-started/cli.mdx

```diff
@@ -19,7 +19,15 @@ The CLI is configured to use a specific FGA server in one of three ways:
 
 - Using CLI flags.
 - Using environment variables.
-- Storing configuration values in a .fga.yaml located in the user’s root directory.
+- Storing configuration values in a .fga.yaml located in one of the following directories, in order of precedence (top has highest priority):
+  - The current working directory
+  - The user-specific config directory: 
+    - On unix systems: $XDG_CONFIG_HOME value if provided, otherwise $HOME/.config
+    - On windows: %AppData%
+  - The fga subdirectory under the user-specific config directory
+  - The current user's home directory:
+    - On unix systems: $HOME
+    - On windows: %USERPROFILE%
```

Comment:
We could make it more clear, sepcific and also improved the formatting!
```suggestion
- Using a `.fga.yaml` configuration file, searched in the following order (highest to lowest priority):

  1. **Current working directory**
  2. **User-specific config directory**

     * Unix: `$XDG_CONFIG_HOME` (if set), otherwise `$HOME/.config`
     * Windows: `%AppData%`
  3. **`fga` subdirectory under the user config directory**
  4. **User's home directory**

     * Unix: `$HOME`
     * Windows: `%USERPROFILE%`
```